cmd/kube-apiserver/apiserver.go

```diff
@@ -193,7 +193,7 @@ func main() {
 		CorsAllowedOriginList:  corsAllowedOriginList,
 		ReadOnlyPort:           *readOnlyPort,
 		ReadWritePort:          *port,
-		PublicAddress:          *publicAddressOverride,
+		PublicAddress:          net.ParseIP(*publicAddressOverride),
```

Comment:
I'm not going to ask you to fix this in this PR, but a followup is OK. 

We can take net.IP as a flag by using util.IP (search for other examples).  We just don't want it in the API types yet.